lib/patch-series.ts

```diff
@@ -868,10 +907,10 @@ export class PatchSeries {
     protected async generateMBox(): Promise<string> {
         const commitRange =
             `${this.project.baseCommit}..${this.project.branchName}`;
-        if (!this.coverLetter &&
-            1 < parseInt(await git(["rev-list", "--count", commitRange],
-                                   { workDir: this.project.workDir }),
-                         10)) {
+        const patchCount =
+            await revListCount(["--no-merges", commitRange],
+                               this.project.workDir);
+        if (!this.coverLetter && 1 < patchCount) {
```

Comment:
Should `this.patchCount` be used here instead of a local?

Reply:
D'oh!